### PR TITLE
fix(deps): rollback cometbft db version upgrade

### PR DESCRIPTION
**Pull request by melekes**
This cometbft-db upgrade is a breaking change which caused zeta database corruption (or at least consensus failures) on their cosmos sdk v0.47.x application when upgrading: zeta-chain/node#3045 (revert with details: zeta-chain/node#3065)

https://github.com/zeta-chain/node/pull/3045
https://github.com/zeta-chain/node/pull/3065

### CODE REVIEW

**Comment by melekes** [comment]
cc @gartnera

File: test/e2e/docker/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
 # instead of spending time compiling them.
-FROM cometbft/cometbft-db-testing:v0.14.1
+FROM cometbft/cometbft-db-testing:v0.12.0
```
Comment (cason) [inline test/e2e/docker/Dockerfile:4]: This is breaking the e2e tests.

**Review by cason — APPROVED** [review]
So, for clearer context, we are downgrading `cometbft-db` on `v0.37.x`, right?

**Comment by melekes** [comment]
> So, for clearer context, we are downgrading `cometbft-db` on `v0.37.x`, right?

correct

**Pull request merged**